spec/factories/intakes.rb

```diff
@@ -238,7 +238,7 @@
   trait :primary_consented do
     primary_consented_to_service_at { 2.weeks.ago }
     primary_consented_to_service { "yes" }
-    primary_consented_to_service_ip { "192.168.2.1" } # IRS approved IP address
+    primary_consented_to_service_ip { "136.62.11.145" } # IRS approved IP address
```

Comment:
The IRS's provided the test IP address on the left exploicitly as one that would "pass". However, it comes back as a reserved IP and fails on submission. 🤔  So I swapped it out with something that doesnt make the IRS complain